tasks/opendkim.yml

```diff
@@ -1,5 +1,12 @@
 ---
 ## Opendkin configuration tasks
+- name: Install epel repository on RHEL based system
+  become: true
+  dnf:
+    name: epel-release
+    state: latest
+  when: 
+    - ansible_os_family == 'RedHat'
```

Comment:
I would rather have separate install files (`install_debian.yml`, `install_redhat.yml`) per platform and conditions during include in `main.yml`

Reply:
Currently, the only thing that differs is the epel installation. Would you rather have install_debian_postfix.yml, install_debian_sendmail.yml, install_debian_opendkim.yml and its redhat equivalent?